code/modules/mob/living/living_say.dm

```diff
@@ -206,6 +206,8 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 		message = "[randomnote] [message] [randomnote]"
 		spans |= SPAN_SINGING
 
+	if(message_mods[WHISPER_MODE]) // whisper away
```

Comment:
Don't do this. See #80980

Reply:
Yeah that's what broke whispering, they moved the mode to the Radio and not the say

Reply:
I meant use LAZYACCESS for your case as well

Reply:
ahh, i missed the word 'right' lol